<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KernelizediFDD设置</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Chinese" country="China"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>active threshold</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spActiveThreshold">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>discover threshold</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>kernel resolution</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spKernelResolution">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>max active base feat</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spMaxActiveBaseFeat">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>max base feat sim</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spMaxBaseFeatSim">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>0.700000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>sparsify</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spSparsify">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spDiscoverThreshold">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
